docs/api.md

```diff
@@ -3931,12 +3978,20 @@ These are in alphabetic order, with the exception that the `Extended` versions o
 + createdAt: `2018-01-19T23:58:03.395Z` (string, required) - ISO date format
 + updatedAt: `2018-03-21T12:45:02.312Z` (string, optional) - ISO date format
 
+## Deleted Form (Form)
++ deletedAt: `2018-03-21T12:45:02.312Z` (string, required) - ISO date format
++ id: `42` (number, required) - Numeric ID as it is represented in the database.
```

Comment:
What do you think about describing `id` in functional terms? Something like:

> Numeric ID that distinguishes the Form from other Forms in the Project with the same `xmlFormId`. This ID can be used to restore the Form.

Reply:
great suggestion!